packages/callhierarchy/src/browser/callhierarchy-contribution.ts

```diff
@@ -28,7 +28,7 @@ export const CALL_HIERARCHY_TOGGLE_COMMAND_ID = 'callhierarchy:toggle';
 export const CALL_HIERARCHY_LABEL = nls.localizeByDefault('Call Hierarchy');
 
 export namespace CallHierarchyCommands {
-    export const OPEN = Command.toLocalizedCommand({
+    export const OPEN = Command.toLocalizedCommand<[], CallHierarchyTreeWidget>({
```

Comment:
In cases like this, it might be extension-friendlier to say the args are `any[]` or `unknown[]` so that if adopters want to add the ability to pass an argument that affects (e.g.) what happens after the widget opens or something, they can register a new handler that accepts an argument without breaking the typing?

Reply:
Defining a handler with different arguments than `[]` should already be possible:

```ts
execute = (newArg1?: A, newArg2?: B, newArg3?: C) => {}
```

All new arguments must be optional as the framework will expect calling the handler with none, as per the current typings.